aboutme.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>More DOM Manipulation</title>
+</head>
+
+<body>
+
+  <h1>About Me</h1>
+
+  <div class="profilePicture"></div>
+
+  <ul>
+    <li class="listitem">Nickname: sumayah <span id="nickname"></span></li>

```

Comment:
You should add the name of the class from JS, not in the index.html
```suggestion
    <li class="listitem">Nickname: sumayah <span id="nickname"></span></li>
```

Reply:
Also, the nickname and favorite animal and hometown should be added from javascript not here.